src/pages/cli/graphql/custom-business-logic.mdx

```diff
@@ -494,6 +494,8 @@ Next, install the AppSync dependencies for your custom resource:
 cd amplify/backend/custom/MyCustomResolvers
 npm i @aws-cdk/aws-appsync@~1.124.0
 ```
+> **Note:** Installing using the '\~' character does not modify the package.json to use '\~' for default npm configurations. Make sure your package.json reflects the right dependency to avoid compatibility errors in CDK.
```

Comment:
```suggestion
> **Note:** Installations using the '\~' character do not modify the package.json. To use '\~' for default npm configurations, make sure your package.json reflects the right dependency to avoid compatibility errors in CDK.
```